src/components/GameRoundPot/GameRoundPot.tsx

```diff
@@ -86,10 +87,15 @@ function GameRoundPot({
     }
 
     if (sanitizeString(tGeo(potCode)) === sanitizeString(currentGuess)) {
-      setCurrentRoundStatus("won");
-      toastSuccess(t("guessedIt"));
+      setTimeout(() => {
+        setCurrentRoundStatus("won");
+        toastSuccess(t("guessedIt"));
+        confetti();
+      }, SQUARE_ANIMATION_LENGTH * squares.length);
     } else if (guesses.length + 1 === maxAttempts) {
-      setCurrentRoundStatus("lost");
+      setTimeout(() => {
+        setCurrentRoundStatus("lost");
+      }, SQUARE_ANIMATION_LENGTH * squares.length);
```

Comment:
toast?

Reply:
As discussed, we probably don’t need it, but might get back to it later.